### ISSUE TITLE
why did file size of images increase?

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/function-builder#115]
From https://github.com/phetsims/resistance-in-a-wire/issues/120#issuecomment-347764684

I said:
> Looking at other sims... There appears to be a general increase in html file size. For example, function-builder 1.0.6 (1358908) to 1.0.7 (1660565), a 22% increase. I'm responsible for this sim, and there were no significant changes that I'm aware of.

@samreid said:
> images/ is 322,761 bytes after https://github.com/phetsims/function-builder/commit/1f90c8067415d297f8e304a91e753fc4c30092ed
images/ is  96,026 bytes in the parent commit which is https://github.com/phetsims/function-builder/commit/20ce3711725c42e8747bcc636ca99ea225c9ae1f
> 
> This is a difference of 226735 bytes, may account for most of the added 301657 bytes (still 74922 bytes unaccounted for).

I said:
> Hmmm... Not sure why that commit added 226735, looks like we were just replacing images with the same dimensions.

**Comment by pixelzoom** [comment]
beaker.png went from `93x123 PNG (32-bit color) 3.00K` to `94x124 PNG (32-bit color) 20.91K`, similar dimensions but almost 7x larger.

**Comment by pixelzoom** [comment]
@amanda-phet FYI.

**Comment by clmccutchan** [comment]
@pixelzoom I'm not sure! But I can check into it. I'm totally slammed with other work until next Tuesday  though.

**Comment by pixelzoom** [comment]
Thanks @clmccutchan. Relatively low priority, so when you have a chance...

**Comment by pixelzoom** [comment]
The increase in size is undoubtedly due to how the files were saved/exported as PNG.  My preferred procedure is to copy each icon to a new .ai file, then "File > Save for Web".  If I do that with the beaker icon, my beaker.png is 2629 bytes, almost 10x smaller than the file created by @clmccutchan.

@clmccutchan how did you create the PNG files?

**Comment by pixelzoom** [comment]
If I use Ilustrator's "File > Export", check "Use artboards", Resolution=300dpi, Anti-aliasing=Art Optimized, then beaker.png is 21638 bytes (21.6K), closer to the 20.91K  that @clmccutchan provided.  So I'm guessing that she used "File > Export" with settings that bloat the file size.

**Comment by clmccutchan** [comment]
@pixelzoom There's a story here. About half of the pngs export out of Illustrator not centered on the artboard. To get the pngs centered, I had to take them into Photoshop and crop them. When I exported the pngs out of Illustrator at lower resolution and then centered them in Photoshop, people thought the final images were too pixelated. So, I exported out of Illustrator at a higher resolution and then centered the offending pngs in Photoshop.

**Comment by pixelzoom** [comment]
@clmccutchan said:
> There's a story here. ...

I see that in [images/cards/README.md](https://github.com/phetsims/function-builder/blob/master/images/cards/README.md), minus the bit about needing to export from Illustrator at higher resolution.

**Comment by samreid** [comment]
> So, I exported out of Illustrator at a higher resolution and then centered the offending pngs in Photoshop.

Photoshop also has a "save for web" option that is compressed.

**Comment by clmccutchan** [comment]
@pixelzoom Just wanted to check in on the status of this. Do you need me to export these at a lower resolution from Photoshop?

**Comment by pixelzoom** [comment]
Thanks for checking in @clmccutchan, I'll take a stab at it.

**Comment by pixelzoom** [comment]
In the preceding commit, I used Photoshop's "Save for Web" to reduce the size of the PNG files.  I opened each PNG file that was checked into GitHub, and simply used "Save for Web" with default settings.  The resulting file sizes are still larger than what we had before we addressed #109 (symmetry problems), but I think that this is acceptable.

Here are the specific of how file size has changed:

filename | before #109 | after #109 | after "Save for Web"
:--- | :--- | :--- | :--- |
beaker.png | 3.00K | 20.91K | 6.78K
butterfly.png | 4.35K | 23.72K | 9.83K
butterfly-hires.png | 16.17K | 43.64K | 29.87K
cherries.png | 5.66K | 23.93K | 10.10K
circle.png | 2.28K | 16.48K | 2.47K
feet.png | 2.52K | 18.92K | 4.53K
planet.png | 7.40K | 26.11K | 12.33K
rectangle.png | 0.21K | 15.14K | 1.07K
snowflake.png | 3.97K | 20.63K | 6.66K
star.png | 2.11K | 16.93K | 2.92K
stickFigure.png | 1.91K | 17.64K | 3.44K
sun.png | 2.44K | 18.84K | 4.33K
triangle.png | 0.95K | 18.54K | 2.00K
**TOTAL** | **69.37K** | **253.86K** | **96.33K**

**Comment by pixelzoom** [comment]
I've tested function-builder and function-builder-basics with the revised images, and I'm confident that this does not introduce any problems that were fixed in #109 and  phetsims/function-builder-basics#18.    

The next step is to push these changes to the 1.0 branch of function-builder. (The 1.1-phetio branch will be ignored, per Slack developer discussion). Then publish maintenance releases of function-builder (1.0 branch) and function-builder-basics (1.0 branch).
